modules/sftmx/components/stats/SftmxStatsFtmStakedFree.tsx

```diff
@@ -0,0 +1,157 @@
+import { useMemo } from 'react';
+import { EChartsOption } from 'echarts';
+import { format } from 'date-fns';
+import { numberFormatUSDValue } from '~/lib/util/number-formats';
+import ReactECharts from 'echarts-for-react';
+import { useBreakpointValue, useTheme } from '@chakra-ui/react';
+import numeral from 'numeral';
+
+interface DataProps {
+    timestamp: number;
+    exchangeRate: string;
+    totalFtmAmount: string;
+    totalFtmAmountInPool: string;
+    totalFtmAmountStaked: string;
+}
+interface Props {
+    data: DataProps[];
+}
+
+export function SftmxStatsFtmStakedFree({ data }: Props) {
+    const { colors } = useTheme();
+    const isMobile = useBreakpointValue({ base: true, lg: false });
```

Comment:
maybe it'll be good to make this usage a common hook in utils somewhere, doesn't have to be in this PR

e.g. // utils.ts
`export const isMobile = () => useBreakpointValue({ base: true, lg: false });`